components/CramerView/src/site/site.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- vim: set tabstop=4 softtabstop=0 expandtab shiftwidth=4 smarttab : -->
+
+<project>
+
+    <skin>
+        <groupId>org.apache.maven.skins</groupId>
+        <artifactId>maven-fluido-skin</artifactId>
+        <version>1.7</version>
```

Comment:
We should use fluido 1.8

Reply:
was autogen'ed when i made the proj so its the only one i vouch for (unless you tell me 1.8 fixes a security bug)